cookbooks/aws-parallelcluster-install/files/default/clusterstatusmgtd/clusterstatusmgtd.py

```diff
@@ -147,12 +147,16 @@ def __str__(self):  # noqa: D105
 
     @staticmethod
     def _transform_compute_fleet_data(compute_fleet_data):
-        compute_fleet_data[
-            ComputeFleetStatusManager.COMPUTE_FLEET_STATUS_ATTRIBUTE
-        ] = ComputeFleetStatus.EVENT_HANDLER_STATUS_MAPPING.value.get(
-            compute_fleet_data.get(ComputeFleetStatusManager.COMPUTE_FLEET_STATUS_ATTRIBUTE), ComputeFleetStatus.UNKNOWN
-        )
-        return compute_fleet_data
+        try:
+            compute_fleet_data[
+                ComputeFleetStatusManager.COMPUTE_FLEET_STATUS_ATTRIBUTE
+            ] = ComputeFleetStatus.EVENT_HANDLER_STATUS_MAPPING.value.get(
+                compute_fleet_data.get(ComputeFleetStatusManager.COMPUTE_FLEET_STATUS_ATTRIBUTE),
+                str(ComputeFleetStatus.UNKNOWN),
+            )
+            return compute_fleet_data
+        except AttributeError:
+            raise
```

Comment:
what is this for? Did you want to add a log message?

Reply:
changed to 
```
        except AttributeError as e :
            raise Exception("Unable to parse compute fleet status data: %s", e)
```